css/style.css

```diff
@@ -0,0 +1,6 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    font-size: 62.5%;
+}
```

Comment:
Add new line at end of file

For why refer this: 
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Resolved